artemis/src/test/resources/complete_config.yaml

```diff
@@ -32,7 +32,7 @@ eth1-endpoint: "http://localhost:8545"
 # logging
 log-color-enabled: True
 log-include-events-enabled: True
-log-destination: "both"
+log-destination: "DEFAULT_BOTH"
```

Comment:
This probably should specify something that's not the default so we can tell it actually got picked up.

Reply:
its tested in other tests, but we can...

Reply:
this test is actually super fragile, i'm going to avoid changing that complete_config file...

Reply:
refactored tests a little bit so that we can see all logging options are picked up from configuration file. I think I've addressed the core of this concern.